marklogic-data-hub-central/ui/e2e/hc-qa-project/step-definitions/custom/customMastering/customMastering.step.json

```diff
@@ -1,18 +1,18 @@
 {
   "lang" : "zxx",
-  "name" : "customMastetring",
+  "name" : "customMastering",
```

Comment:
Let's not call this "customMastering", because it's not doing any mastering. It's just a vanilla custom step. The same applies for the other custom non-ingestion steps - they're just vanilla steps. I don't want to propagate the notion that there's some concept in DHF of a "custom mastering" step - there's not. From DHF's perspective, a custom step is simply just that - a custom step. 

It looks like custom custom/mapping/mastering/matching/merging are all identical, in which case I recommend getting rid of 4 of them and just have a single custom step.

Reply:
Its a place holder for all the different custom steps possible and to add custom code appropriate for custom matching/merging/mastering/mapping

Reply:
That's my point though - there aren't different types of custom steps, there's just "a custom step". What the code does doesn't matter to DHF. By calling it a "custom mapping" or "custom matching" step, that suggests that DHF will do something based on it being related to mapping or matching. But DHF won't. All DHF will do is run the step module.

Reply:
Correct. I am with you on that. We want to run an e2e scenario where DHF will run the step module for mapping, matching etc. We dont have the code yet for these step module and hoping we can get that in the upcoming hardening week, where engineers will test using a step module for mapping, matching etc